Crashlytics/Crashlytics/Rollouts/StringToHexConverter.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+// This is a swift rewrite for the logic in FIRCLSFile for the function FIRCLSFileHexEncodeString()
+public func stringToHexConverter(for string: String) -> String {
```

Comment:
(just curious) Is it worth writing any unit tests for this method itself (other than the way it's tested in CrashlyticsRemoteConfigManagerTests.swift)

Reply:
I can probably writing a test in `FIRCLSFileTests` to compare the output between `FIRCLSFileHexEncodeString()` and `stringToHexConverter`

Reply:
Added unit tests under `Crashlytics/UnitTests/FIRCLSFileTests.m`